backend/src/services/taskService.ts

```
@@ -100,11 +166,31 @@ export class TaskService {
 
-    // Resolve schedule_id: use provided value, then fall back to user's active/default schedule
-    // Uses a single optimized query instead of 4 sequential queries
+    // Resolve schedule_id: use provided value, else use project default schedule if configured,
+    // otherwise fall back to user's active/default schedule.
+    // Uses a single optimized query instead of 4 sequential queries.
     const scheduleStart = Date.now();
     let scheduleId = input.schedule_id;
 
+    if (!scheduleId && input.project_id) {
+      try {
+        const { data: project } = await client
```

Biome (2.4.6)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)